locations/paas/src/main/java/brooklyn/location/PaasProvisioningLocation.java

```diff
@@ -0,0 +1,22 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package brooklyn.location;
+
+public interface PaasProvisioningLocation<T extends PaasLocation> extends ProvisioningLocation<T> {
```

Comment:
We also addressed that issue. When using CF, there's no need (apparently) to provision the application container prior to deployment and it's all executed in one step. However, provisioning (obtain/release) could be seen as creating/instantiating a connection with the CLI. 

WDYT?